rpcd/playbooks/roles/elasticsearch/templates/legacy-mapping.sh.j2

```diff
@@ -0,0 +1,302 @@
+#!/bin/sh
+curl --retry 10 -XPUT 'http://{{ hostvars[inventory_hostname]['container_address'] }}:{{ elasticsearch_http_port }}/_template/legacy-liberty' -d '
+{
+  "template" : "*",
+  "order": 0,
+    "mappings" : {
+      "_default_" : {
+        "dynamic_templates" : [ {
```

Comment:
Not a huge deal but seeing `[ {` and later `} ]` isn't great since the rest of the indentation implies that all bracing will be accompanied by indents, but knowing this is a list of dicts means we have to look extra hard to make sure both `[` and `{` are being closed properly. I realize there's only one dynamic template here (and the other occurrence of this a bit below), but maybe consider breaking the list and dict start out and indenting both accordingly.